tests/integration/ceph_object_test.go

```diff
@@ -403,46 +403,47 @@ func testObjectStoreOperations(s suite.Suite, helper *clients.TestClient, k8sh *
 	t.Run("delete OBC", func(t *testing.T) {
 		i := 0
 		dobcErr := helper.BucketClient.DeleteObc(obcName, bucketStorageClassName, bucketname, maxObject, true)
-		assert.Nil(s.T(), dobcErr)
+		assert.Nil(t, dobcErr)
 		logger.Info("Checking to see if the obc, secret and cm have all been deleted")
 		for i = 0; i < 4 && !helper.BucketClient.CheckOBC(obcName, "deleted"); i++ {
 			logger.Infof("(%d) obc deleted check, sleeping for 5 seconds ...", i)
 			time.Sleep(5 * time.Second)
 		}
-		assert.NotEqual(s.T(), 4, i)
+		assert.NotEqual(t, 4, i)
 
 		logger.Info("ensure OBC bucket was deleted")
 		var rgwErr int
-		for i = 0; i < 4; i++ {
+		retries := 6
+		for i = 0; i < retries; i++ {
 			_, rgwErr, _ = rgw.GetBucket(rgwcontext, bucketname)
 			if rgwErr == rgw.RGWErrorNotFound {
 				break
 			}
 			logger.Infof("(%d) check bucket deleted, sleeping for 5 seconds ...", i)
 			time.Sleep(5 * time.Second)
 		}
-		assert.NotEqual(s.T(), 4, i)
-		assert.Equal(s.T(), rgwErr, rgw.RGWErrorNotFound)
+		assert.NotEqual(t, retries, i)
+		assert.Equal(t, rgwErr, rgw.RGWErrorNotFound)
```

Comment:
For whatever reason, this seems to take slightly longer than 20 seconds on our k8s 1.16 tests. So I edit the test to make sure things are passing.

The `s.T()` --> `t` changes make the assert failures cause the right subtest to fail. That wasn't happening and was causing confusing output during failures.